src/termwidget.cpp

```diff
@@ -254,6 +286,15 @@ TermWidget::TermWidget(TerminalConfig &cfg, QWidget * parent)
     setLayout(m_layout);
 
     m_layout->addWidget(m_term);
+    const auto objs = m_term->children();
+
+    for (QObject *o : objs)
+    {
+        // Find TerminalDisplay
+        if (!o->isWidgetType() || qobject_cast<QWidget*>(o)->isHidden())
+            continue;
```

Comment:
Ditto for curly brackets. Indeed the original code doesn't include brackets. Let's improve the code when you're at it.